pkg/querier/tenantfederation/merge_queryable_test.go

```diff
@@ -264,18 +276,19 @@ type mergeQueryableScenario struct {
 	doNotByPassSingleQuerier bool
 }
 
-func (s *mergeQueryableScenario) init() (storage.Querier, error) {
-	// initialize with default tenant label
-	q := NewQueryable(&s.queryable, !s.doNotByPassSingleQuerier, defaultConcurrency, log.NewNopLogger())
-
+func (s *mergeQueryableScenario) init(t *testing.T) (context.Context, storage.Querier) {
 	// inject tenants into context
 	ctx := context.Background()
 	if len(s.tenants) > 0 {
 		ctx = user.InjectOrgID(ctx, strings.Join(s.tenants, "|"))
 	}
+	// initialize with default tenant label
```

Comment:
nit: moving it just made the diff bigger, otherwise no effect

Reply:
Yep, you're right. I'll undo this change.

Reply:
Done.